lib/clowne/adapters/active_record/dsl.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Clowne
+  module Adapters
+    # Extend ActiveRecord with Clowne DSL and methods
+    module ActiveRecordDSL
+      module InstanceMethods
+        # Shortcut to call class's cloner call with self
+        def clowne(*args)
```

Comment:
Not sure about the naming. I don't like amoeba style (i.e. `clowne_dup`) – too verbose; from the other hand, `clowne` could be confused with `clone`.

Reply:
IMHO its our "special word" and is well suited for this case :+1: